### PR TITLE
12088 Imported latest sbc-common-components

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#12088

*Description of changes:*
- imported latest sbc-common-components (2.4.46)
- app version = 5.0.2

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
I found a bug in the imported footer, so I'll fix that first and then import the latest library here once only ;)

File: package-lock.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
         "provinces": "^1.11.0",
         "regenerator-runtime": "^0.13.7",
         "register-service-worker": "^1.7.1",
-        "sbc-common-components": "2.4.38",
+        "sbc-common-components": "2.4.46",
```
Comment (severinbeauvais) [inline package-lock.json:32]: I had a look at all the possible breaking changes from 38 -> 46 according to this:
https://github.com/bcgov/sbc-common-components/releases

It all looks fine :)

File: package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "provinces": "^1.11.0",
     "regenerator-runtime": "^0.13.7",
     "register-service-worker": "^1.7.1",
-    "sbc-common-components": "2.4.38",
+    "sbc-common-components": "2.4.46",
```
Comment (severinbeauvais) [inline package.json:39]: I had a look at all the possible breaking changes from 38 -> 46 according to this:
https://github.com/bcgov/sbc-common-components/releases

It all looks fine :)

File: src/components/dialogs/ConfirmDissolution.vue [hunk 3]
```diff
@@ -12,7 +12,8 @@
         <p class="warning-title" id="dialog-title">
         {{getModalTitle}}
         </p>
-        <p class="warning-text" id="dialog-text">You are about to voluntarily dissolve <strong>{{ getEntityName }}</strong>;
+        <p class="warning-text" id="dialog-text">
+          You are about to voluntarily dissolve <strong>{{ getEntityName }}</strong>;
```
Comment (severinbeauvais) [inline src/components/dialogs/ConfirmDissolution.vue:16]: Lint fix (line was > 120 columns). Same in other file below.

**Comment by severinbeauvais** [comment]
Tested locally. Looks fine.

image

**Review by lewischenstudio — APPROVED** [review]

**Review by lewischenstudio — APPROVED** [review]

**Pull request merged**